src/main/java/com/lothrazar/cyclic/block/solidifier/TileSolidifier.java

```diff
@@ -71,6 +71,10 @@ public void tick() {
       this.timer = 0;
       return;
     }
+    //Fixes sync issue entirely
+    if(level.isClientSide()) return;
+    //Checks if there is space in the output slot
+    if(this.outputSlots.getStackInSlot(0).getCount() > 64 - currentRecipe.getResultItem(level.registryAccess()).getCount()) return;
```

Comment:
can we assume the max is always 64? what if its an ender pearl and 16 is the max. well we can check that later, still a good fix